### ISSUE TITLE
Issue with lists on section landing pages

### ISSUE THREAD

**Issue by alexearnshaw** [google/docsy#169]
In our documentation site that uses Docsy, we've noticed that the list of links that appear on section landing pages do not point to the correct URLs on a Netlify deploy preview - they link to the production site instead of the deploy preview site. The links in the left navigation correctly link to the deploy preview. 

For an example see this issue in our Github project https://github.com/Axway/axway-open-docs/issues/214

I'm not sure if this is an issue with the Docsy theme, or with Hugo, or if it's something we haven't configured correctly on our side...would anyone be able to advise on what differs between how the URLs in the left nav are built and how the URLs in the section list are built,  and/or suggest a possible fix?

Thanks a lot :)

**Comment by LisaFC** [comment]
I'll take a look!

**Comment by alexearnshaw** [comment]
Great, appreciate anything that will set me in the right direction to fix :)

**Comment by LisaFC** [comment]
So I think it's something to do with how Netlify deals with baseURL - the Docsy site itself doesn't have a baseURL set and all the links are fine in Netlify deploy previews (as in they're all to the preview version). Why it's using the baseURL in one set of links and not in the other continues to be mysterious, as they both just use .RelPermalink.

This might fix it, let me know if it works!
https://discourse.gohugo.io/t/correct-baseurl-on-netlify-previews/10429

Though you'll probably need to do the full weirdy build command that gets the submodules so something like:
"cd themes/docsy && git submodule update -f --init && cd ../.. && hugo -b $DEPLOY_PRIME_URL"

**Comment by alexearnshaw** [comment]
Hmm, thanks @LisaFC that's interesting. There's no particular reason that I know of as to why we have baseURL set in our config.toml so I'm thinking I might try setting it to the same as Docsy first,  e.g. 
`baseURL = "/"`

If that doesn't work, I'll try the build command with $DEPLOY_PRIME_URL

Will let you know if either of those work

**Comment by alexearnshaw** [comment]
Ok so just to report back...Changing baseURL to "/" did not work for us as although it fixed the section links in the page body it broke the links we've added in the right navigation (as we use baseURL for them).

Using $DEPLOY_PRIME_URL in the build command for deploy previews worked for both types of links so this is what we went with in the end. We ended up using a `netlify.toml` file so that we could pass DEPLOY_PRIME_URL to the build command for previews, and not pass it for production builds. Our settings can be seen in https://github.com/Axway/axway-open-docs/pull/242 if it's helpful to anyone.

Thanks @LisaFC for the assistance, this can be closed now :)
